### PR TITLE
[SM-810] remove token permission UI

### PR DESCRIPTION
**Pull request by willmartian**
## Type of change

```
- [x] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Removes UI that alludes to individual Access Token permissions, since those don't exist in the product. 

This PR also removes the "placeholder" text where the SA name should be. 

## Code changes

- **file.ext:** Description of what was changed and why

## Screenshots

Removed permission form field in dialog:

image

Removed permission column in table:

image

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

**Review by coltonhurst — COMMENTED** [review]
Looking good, just one comment.

File: bitwarden_license/bit-web/src/app/secrets-manager/service-accounts/access/dialogs/access-token-create-dialog.component.ts [hunk 1]
```diff
@@ -8,13 +8,15 @@ import { BitValidators } from "@bitwarden/components";
...
+import { ServiceAccountService } from "../../service-account.service";
 import { AccessService } from "../access.service";
 
 import { AccessTokenDetails, AccessTokenDialogComponent } from "./access-token-dialog.component";
 
 export interface AccessTokenOperation {
   organizationId: string;
-  serviceAccountView: ServiceAccountView;
+  serviceAccountId: string;
+  // serviceAccountView: ServiceAccountView;
```
[the remark this answers is not in the document]
Comment (willmartian) [reply bitwarden_license/bit-web/src/app/secrets-manager/service-accounts/access/dialogs/access-token-create-dialog.component.ts]: What comment? (Removed in [241e978](https://github.com/bitwarden/clients/pull/5681/commits/241e9788eb512fd445b81e9446f63caa515b6287))

**Review by coltonhurst — COMMENTED** [review]
Ok, one more small thing. (This may be caused by something else and we need to make a separate ticket as a follow up, but I thought it was worth mentioning.) Please note I am running Mac OS with the Firefox Developer's Edition browser.

On the `master` branch, this is the animation when opening the dialog box to create an access token:

https://github.com/bitwarden/clients/assets/5691612/63d983c9-82c0-4017-9d08-54ce1a673cdd

This is the animation after this change:

https://github.com/bitwarden/clients/assets/5691612/d0a34dce-7a18-4a21-8c52-eb9bb17f2132

The opening animation is not fluid anymore, with the small dialog flash in the middle of the opening animation. Are you able to reproduce this?

**Comment by willmartian** [comment]
@coltonhurst Good catch. That was introduced in [1ce6f38](https://github.com/bitwarden/clients/pull/5681/commits/1ce6f38fd69be59d1d069d368e120b1841df3834). The resolution was to either add a loading spinner to the component, or pass the service account name in during the creation of the dialog. I went with the latter [here](https://github.com/bitwarden/clients/pull/5681/commits/36e20460bbe2f1bcc32933d3e3307220e84ba4a8#diff-7c832810cdb56b642844942cc2de0e85a2e938be4fef8873647d051e148c51b8R80).

**Review by Thomas-Avery — DISMISSED** [review]
LGTM

**Pull request merged**